src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -85,28 +88,29 @@ internal virtual void PackFromXyzBytes(BufferSpan<byte> sourceBytes, BufferSpan<
         /// <summary>
         /// Bulk version of <see cref="IPixel.ToXyzBytes(byte[], int)"/>.
         /// </summary>
-        /// <param name="sourceColors">The <see cref="BufferSpan{T}"/> to the source colors.</param>
-        /// <param name="destBytes">The <see cref="BufferSpan{T}"/> to the destination bytes.</param>
+        /// <param name="sourceColors">The <see cref="Span{T}"/> to the source colors.</param>
+        /// <param name="destBytes">The <see cref="Buffer{T}"/> to the destination bytes.</param>
+        /// <param name="startIndex">The starting index of the <paramref name="destBytes"/>.</param>
         /// <param name="count">The number of pixels to convert.</param>
-        internal virtual void ToXyzBytes(BufferSpan<TPixel> sourceColors, BufferSpan<byte> destBytes, int count)
+        internal virtual void ToXyzBytes(Span<TPixel> sourceColors, Buffer<byte> destBytes, int startIndex, int count)
```

Comment:
Still, I don't think it's a good API in it's current form. 
1. As described in this issue it can't be used with `Span<T>` without having a perf loss.
2. There are [several](https://github.com/JimBobSquarePants/ImageSharp/blob/master/src/ImageSharp/Quantizers/OctreeQuantizer%7BTPixel%7D.cs#L29) [places](https://github.com/JimBobSquarePants/ImageSharp/blob/master/src/ImageSharp/Formats/Png/PngEncoderCore.cs#L544) where we are allocating 4-sized arrays to extract RGBA values.

I wanted to propose changing this for a long time. I want to use this occasion, and @JimBobSquarePants's fresh head, so it's time for a gist! ;)